doc/source/parameters/unresolved-cfd-dem/void-fraction.rst

```diff
@@ -33,6 +33,7 @@ If the ``mode`` chosen is ``pcm``, then the void fraction is calculated using th
   set quadrature rule              = gauss
   set n quadrature points          = 0
   set project particle velocity    = false
+  set project particle forces      = false
```

Comment:
It feels weird to put the parameter there. All of the force models are parametrized from the cfd-dem section and not the void fraction section. I think this parameter should be move to the cfd-dem section where the forces are parametrized.